src/main/java/de/tum/in/www1/artemis/service/FileService.java

```diff
@@ -1224,14 +1222,16 @@ public void deleteFiles(List<Path> filePaths) {
      * @param fileName        file name to set file name
      * @param extension       extension of the file (e.g .pdf or .png)
      * @param streamByteArray byte array to save to the temp file
-     * @return multipartFile wrapper for the file stored on disk
+     * @return multipartFile wrapper for the file stored on disk with a cleared name
```

Comment:
```suggestion
     * @return multipartFile wrapper for the file stored on disk with a sanitized name
```
Why use two different words?